dashboards/google-kubernetes-engine/gke-compute-resources-cluster-view.json

```diff
@@ -30,6 +30,7 @@
         "widget": {
```

Comment:
Upon further thought I think all the charts in the first row need updates. There are two valid versions of this first chart:
- container cpu usage / node _allocatable_ cores (user space cpu for both numerator and denominator)
- node cpu usage / node _total_ cores (user+system cpu for numerator and denominator)

The current calculation could erroneously go >100% as I understand these metrics.

Reply:
Yes, I think [you're right](https://learnk8s.io/allocatable-resources).

I went for the second option to be consistent with what we report in the CPU / Memory charts of GKE OiC
![Screenshot 2023-02-27 at 5 18 19 PM](https://user-images.githubusercontent.com/6540967/221698795-21e76e01-5b34-4e52-942e-311cdcfbf3b8.png)